mobile/ios/Runner/Info.plist

```diff
@@ -76,5 +76,16 @@
     <false />
     <key>CADisableMinimumFrameDurationOnPhone</key>
     <true />
+    <key>CFBundleURLTypes</key>
```

Comment:
Also, Is this something specific to using OAuth?

Reply:
Same as before but for iOS, don't know if it works tho

Reply:
I can test this part when we have this ready.

Reply:
That would be awesome, it should already work if you want to test it. I just need to verify the behavior with local accounts

Reply:
When you have a chance, can you write down some steps on how to test and the expected behavior of the test? 

Also, there will be quite a refactoring work when #206 is merged into master

Reply:
I don't have much time in the next days but of course I will!

Yeah I know about that refactoring, I changed a lot of things too 😬